twinkle.js

```diff
@@ -110,6 +110,8 @@ Twinkle.defaultConfig = {
 	autoMenuAfterRollback: false,
 
 	// XfD
+	logXfdNominations: false,
+	xfdLogPageName: 'XFD log',
```

Comment:
Suggest `XfD log` instead

Reply:
Went back and forth on it, never know how many folks prefer the (over-?)stylized version, but I do.  It doesn't help that Twinkle is somewhat inconsistent, although we do lean toward XfD.

Reply:
Folks who got back to me felt the same as us, so changed!